ts-defs-src/test-api/request-hook.d.ts

```diff
@@ -203,6 +203,10 @@ interface RequestOptions {
      * `proxyAuth`, `authHeader` and `bypassRules`.
      */
     proxy: Record<string, unknown>;
+    /**
+     * Specifies whether the request is AJAX (xhr or fetch).
```

Comment:
AJAX is not an adjective :(

Reply:
That's a shame(

> Specifies whether the request is an AJAX request (xhr or fetch).

Reply:
https://docs.microsoft.com/en-us/dotnet/api/system.web.mvc.ajaxrequestextensions.isajaxrequest?view=aspnet-mvc-5.2